python-package/xgboost/sklearn.py

```diff
@@ -258,7 +268,8 @@ def fit(self, X, y, sample_weight=None, eval_set=None, eval_metric=None,
         if evals_result:
             for val in evals_result.items():
                 evals_result_key = list(val[1].keys())[0]
-                evals_result[val[0]][evals_result_key] = val[1][evals_result_key]
+                evals_result[val[0]][evals_result_key] = val[
```

Comment:
This was formatted by autopep8. If i put it in one line, it will exceed 80 character line width. This file already has so many pep8 suppressing commands. Do you still want me to put it in one line?